robolectric/src/main/java/org/robolectric/internal/bytecode/InstrumentationConfiguration.java

```diff
@@ -120,6 +126,20 @@ public InstrumentationConfiguration build() {
           ShadowedObject.class,
           TempDirectory.class
       ));
+      packagesToNotAquire.addAll(Arrays.asList(
+          "java.",
+          "javax.",
+          "sun.",
+          "com.sun.",
+          "org.w3c.",
+          "org.xml.",
+          "org.junit",
+          "org.hamcrest",
+          "org.specs2",  // allows for android projects with mixed scala\java tests to be
+          "scala.",      //  run with Maven Surefire (see the RoboSpecs project on github)
+          "kotlin.",
+          "com.almworks.sqlite4java" // Fix #958: SQLite native library must be loaded once.
```

Comment:
I think so, but guys requested to keep behaviour exactly the same as before, and it didn't have a dot. I guess this will require separate bug report, if you run into problems with it.